src/rapids_dependency_file_generator/rapids_dependency_file_generator.py

```diff
@@ -201,9 +235,13 @@ def get_filename(file_type, file_prefix, matrix_combo):
     file_ext = ""
     if file_type == str(OutputTypes.CONDA):
         file_ext = ".yaml"
-    if file_type == str(OutputTypes.REQUIREMENTS):
+    elif file_type == str(OutputTypes.REQUIREMENTS):
         file_ext = ".txt"
         file_type_prefix = "requirements"
+    elif file_type == str(OutputTypes.PYPROJECT):
+        file_ext = ".toml"
+        # Always hardcoded for this output type.
```

Comment:
Does this comment still make sense?

Reply:
Yes. For requirements.txt or conda env files dfg prefixes the file name with the key from the `files` section of dependencies.yaml (e.g. `all_cuda*`), but this doesn't make sense for pyproject.toml files. I will update the comment to describe this more explicitly.

Reply:
Oh I see what you mean. Yes, I will update both the README and this comment accordingly.